documentation/src/main/asciidoc/topics/proc_xsite_configuring_state_transfer.adoc

```diff
@@ -0,0 +1,30 @@
+[id='configure_state_transfer-{context}']
+= Configuring Cross-Site State Transfer
+
+Configure the state transfer parameters and mode.
+
+.Procedure
+* Use `<state-transfer>` tag to configure the state transfer parameters and mode.
+
+.Example
+[source,xml,options="nowrap",subs=attributes+]
+----
+include::config_examples/xsite_state_transfer.xml[]
+----
+
+* `chunk-size`: The number of cache entries to batch in each transfer.
+* `timeout`: The time, in milliseconds, to wait for the backup site acknowledge the state chunk received and applied.
+* `max-retires`: Sets the maximum number of retry attempts for push state failures.
+Specify a value of 0 (zero) to disable retry attempts.
+* `wait-time`: The time, in milliseconds, to wait between retry attempts for push state failures.
+The value must be greater than 0 (zero).
+* `mode`: Possible values are `MANUAL` and `AUTO`.
+`AUTO` is only available for `ASYNC` backup strategy and starts the state transfer automatically when a connection
+to a remote site is established.
+If `MANUAL`, the state transfer must be triggered manually.
```

Comment:
@pruivo Can we please delete the list of state transfer configuration properties here? This is reference info and is available in the schema.

Reply:
sure